networkit/cpp/centrality/test/CentralityGTest.cpp

```diff
@@ -501,6 +501,49 @@ TEST_P(CentralityGTest, testPageRank) {
     doTest(PageRank::Norm::L2Norm);
 }
 
+TEST_P(CentralityGTest, testNormalizedPageRank) {
+    SNAPGraphReader reader(isDirected());
+    auto G = reader.read("input/wiki-Vote.txt");
+
+    auto doTest = [&G](PageRank::Norm norm) {
+        PageRank pr(G, 0.85, 1e-8, true);
+        pr.norm = norm;
+        pr.run();
+
+        auto pr_ranking = pr.ranking();
+        constexpr double eps = 1;
+        if(norm == PageRank::Norm::L1Norm) {
+            if (G.isDirected()) {
+                constexpr double eps = 1e-2;
+            EXPECT_EQ(pr_ranking[0].first, 326);
+            EXPECT_NEAR(pr_ranking[0].second, 133.1727, eps);
```

Comment:
Indentation here is wrong. Also, I don't think that using `constexpr` here is helpful, just use `const` (the same applies in other parts of this function).

Reply:
Also, avoid to re-declare `eps`. Can't we test normalized and non-normalized results with the same precision?

Reply:
We can't test both tests with the same precision because of the difference in the magnitude of results and EXPECT_NEAR uses an absolute difference between the results.